libp2p/src/main/kotlin/io/libp2p/mux/yamux/YamuxFrame.kt

```diff
@@ -8,16 +8,18 @@ import io.netty.buffer.Unpooled
 
 /**
  * Contains the fields that comprise a yamux frame.
- * @param streamId the ID of the stream.
+ * @param id the ID of the stream.
  * @param flag the flag value for this frame.
+ * @param length the length field for this frame.
  * @param data the data segment.
  */
-class YamuxFrame(val id: MuxId, val type: Int, val flags: Int, val lenData: Long, val data: ByteBuf? = null) :
+class YamuxFrame(val id: MuxId, val type: Int, val flag: Int, val length: Long, val data: ByteBuf? = null) :
```

Comment:
I believe the field is still `flags`. Though I'm not sure there are use cases when more than one flag is set. 
https://github.com/hashicorp/yamux/blob/master/spec.md#framing

Reply:
You are right. Fixed it.